webpack/assets/javascripts/react_app/components/common/forms/validators.js

```diff
@@ -0,0 +1,14 @@
+import { translate as __ } from '../../../../react_app/common/I18n';
+
+export const required = (value) => value ? undefined : __("can't be blank");
+
+export const maxLength = (number) => (value) => value.length <= number ? undefined : __("too long");
```

Comment:
this should be identical to rails messages (for consistenticy across forms) see https://github.com/theforeman/foreman/blob/c599de355551b7756a683f202d894a9ebf7c7fa4/webpack/assets/javascripts/react_app/common/reduxFormI18n.js (which I guess should be removed?)